packages/react-core/src/components/Alert/__tests__/AlertActionLink.test.tsx

```diff
@@ -20,6 +20,18 @@ test('Renders children', () => {
   expect(screen.getByRole('button')).toBeVisible();
 });
 
+test('Renders ReactNode as children', () => {
+  render(
+    <AlertActionLink>
+      <a href="https://test.link.com/" target="_blank" rel="noreferrer" title="Learn more link">
+        Learn more
+      </a>
+    </AlertActionLink>
+  );
+
+  expect(screen.getByRole('link')).toBeVisible();
+});
```

Comment:
Even though it's just a test, I feel like it'd be better to test a structure we'd recommend/is valid. Nesting interactive content like a link inside a button isn't valid and will throw an a11y error in Axe

Reply:
I agree, I used the code snippet provided by author of the issue for the test.